以下のブロックを変更

ariakira opened the pull request:
#40 に関連して以下を追加しました

1. tips→tipに変更(fix typo)
2. noteを追加
3. embedを追加

Commit:
以下のブロックを変更

1. tips→tipに変更(fix typo)
2. noteを追加
3. embedを追加

Commit:
fix block_embed

vvakame reviewed (changes requested):
PRありがとうございます

### 1. tip への変更 

https://github.com/kmuto/review/blob/232f9925c55638607924e5c3fee98b20aef52654/doc/format.ja.md#%E5%9B%B2%E3%81%BF%E8%A8%98%E4%BA%8B ですね
よさそうです。ありがとうございます。

### 2. noteを追加

別途コメント書きました(大枠でよさそう

### 3. embedを追加

https://github.com/kmuto/review/blob/232f9925c55638607924e5c3fee98b20aef52654/doc/format.ja.md#%E7%94%9F%E3%83%87%E3%83%BC%E3%82%BF%E8%A1%8C によると、rawとは結構仕様が違いそうですね…。
少なくとも、 format.ja.md に記載のパターンが網羅されたテストがあると嬉しいです。
test/fixture/valid/block_embed/content.re
test/fixture/valid/inline_embed/content.re
このあたりかな…？

Commit:
以下の並びをもとにblock_info,block_note関連のコード並び順を変更
https://github.com/kmuto/review/blob/232f9925c55638607924e5c3fee98b20aef52654/doc
/format.ja.md#%E5%9B%B2%E3%81%BF%E8%A8%98%E4%BA%8B
